### ISSUE TITLE
help me in improving accuracy

### ISSUE THREAD

**Issue by balajib363** [serengil/tensorflow-101#1]
I followed your ideas to train a model. After training when I test with my own data my predictions are not upto the mark.
Please share me your accuracy level.
print('Test accuracy:', 100*score[1])

**Comment by serengil** [comment]
You are asking this for which source file? Would you mention the file name directly?

**Comment by balajib363** [comment]
facial-expression-recognition.py
dont mind for not mentioning the file name.

**Comment by serengil** [comment]
I've got the following metrics
Test loss: 2.279457058505316
Test accuracy: 57.42546670713874

BTW, you can directly use the pre-trained weights stored in the path: https://github.com/serengil/tensorflow-101/tree/master/model . Set fit variable false in the main program and put all those files in same directory.

**Comment by BalajiB3663** [comment]
Great job. I deployed with your models, it as good accuracy. But for better understanding just want to check can we improve our accuracy by tuning some hyper parameter like increasing learning epochs or any better ideas.
Thanks for all your response 🙂

**Comment by serengil** [comment]
Actually, you should re-design the model (add new convolution, pooling layers). But it requires test and test again. Also, you should train with gpu because it lasts very long. 

BTW, kaggle competition winner model got %34 accuracy. Here, we've already imporved that accuracy (%57). I mentioned it in related blog post: https://sefiks.com/2018/01/01/facial-expression-recognition-with-keras/

**Comment by BalajiB3663** [comment]
Nice job.
Thanking you for quick clarification on related work.

Please can close the issue.

**Comment by BalajiB197** [comment]
Hi,

Interestingly working on your code for sentimental analysis.
In your emotion-analysis-from-video.py project, when I pass the image
frames within a tensorflow session, model.predict() is not returning any
values. Its constant for all emotions 14.2, 14.3, etc something in that.
I am interested in knowing why this weird thing is happening.
I am working on it for it for 2 continuous days.

Thanks,
Balaji B
